scripts/diff_l10n_branches.py

```diff
@@ -39,7 +39,7 @@
 ...
 # check what is updated in the upstream 
 git diff {{ l_commit }} {{ r_commit }} -- {{ files_to_be_modified.0.filepath }}
-# apply changes to content/ko
+# apply changes to {{ l10n_lang_path }}
```

Comment:
This is a comment, so `{{ l10n_lang_path }}` won't expand, but it still makes sense to me.
`content/ko` is definitely not right.

Reply:
Ah no, it will expand! Neat.